erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md

```diff
@@ -77,8 +77,8 @@ Then the emails sent would look like this:
 Status indicates the state of the Email Campaign, the various Statuses are:
 
 - **Scheduled**: When the Email Campaign has not yet started but scheduled on a future 'Start Date'.
-- **In Progress**: When there are still more campaign mails to be sent.
-- **Completed**: When all the emails have been sent according to the schedule set up in the Campaign.
+- **In Progress**: When the Email Campaign is in progress.
```

Comment:
The previous statement is more explanatory. Of course, the user would understand that 'In progress' for an Email campaign would mean that the campaign is in progress. But it means that some emails are sent and some are left to be sent.

Reply:
Made the changes discussed